samples/popup/popup.cpp

```diff
@@ -216,6 +218,9 @@ void SimpleTransientPopup::OnMouse(wxMouseEvent &event)
     rect.SetWidth(1000000);
     wxColour colour(*wxLIGHT_GREY);
 
+    if ( wxSystemSettings::GetAppearance().IsDark() )
+        colour.Set(90, 90, 90); // dark grey
+
```

Comment:
Sorry for being so annoying, but could we use `SelectLightDark()` here too please?

Reply:
You have nothing to apologize for, it's me who has to apologize for being so sloppy.